storey/dataframe.py

```diff
@@ -95,3 +97,64 @@ async def _do(self, event):
         else:
             df = event.body
             df.to_parquet(path=self._path, partition_cols=self._partition_cols)
+
+
+class FramesClient:
```

Comment:
why do we even need this? why can't the ToTSDB step just have a `v3io_frames.Client` object directly